### PR TITLE
added a function to report info of uploaded samples to gensam

### PR DESCRIPTION
**Pull request by sima-r**

### CODE REVIEW

File: weekly_report.py [hunk 1]
```diff
@@ -38,7 +41,11 @@
               help='Send results as e-mail')
 @click.option('--debug', is_flag=True,
               help='Print debug information.')
-def main (logdir, articdir, nextseqdir, eurofinsdir, outfile, send_email, config, debug, converter):
+@click.option('--uploadedsamples', default='/medstore/results/clinical/SARS-CoV-2-typing/nextseq_data/gensam_upload/reports/Samples_to_gensam.csv',
```
Comment (pbiology) [inline weekly_report.py:44]: lower case 's' please

File: weekly_report.py [hunk 2]
```diff
@@ -198,12 +205,15 @@ def main (logdir, articdir, nextseqdir, eurofinsdir, outfile, send_email, config
 
     #Write the data from nextseq
     write_nextseq(inhouse_dict, outf, logger, conf_dict)
+
+    #Write data  based on uploaded samples to FHM
+    write_uploaded_samples(uploadedsamples, outf)
+
```
Comment (pbiology) [inline weekly_report.py:211]: Put this after eurofins and notes

File: weekly_report.py [hunk 3]
```diff
@@ -476,5 +487,41 @@ def setup_logger(name, log_path=None):
...
+def write_uploaded_samples(uplodedSamplesFile, outf):
+    outf.write('Uploaded to Gensam \n')
+    Samples= pd.read_csv(uplodedSamplesFile, sep=',')
+    df = Samples.groupby(['selection_criteria','upload_week','sequencing_week']).size().reset_index(name='counts')
+    data = {}
+    weeks= df['sequencing_week'].unique().tolist()
+    for week in weeks:
+        data[week] ={'general':0,'unknown': 0, 'selection': 0, 'upload_week': 'not given', 'total' : 0}
+    for index,row in df.iterrows():
+        if row['selection_criteria'] <= 10:
```
Comment (pbiology) [inline weekly_report.py:499]: Change to 99 so we can have more things coming in, or is integer

File: weekly_report.py (hunk 2 above)
Comment (sima-r) [reply weekly_report.py:211]: aa5b84b

File: weekly_report.py (hunk 3 above)
Comment (sima-r) [reply weekly_report.py:500]: 959e53d1be91043505ac3fbd6d6b5648df6ca248

**Pull request merged**